dart/lib/src/sentry_tracer.dart

```diff
@@ -70,10 +87,11 @@ class SentryTracer extends ISentrySpan {
     String operation, {
     String? description,
   }) {
-    return _rootSpan.startChild(
+    final child = _rootSpan.startChild(
```

Comment:
any reason to not return this directly?

Reply:
Also an artefact of adding and then removing code.